root/static/scripts/edit/URLCleanup.js

```diff
@@ -539,6 +539,42 @@ const CLEANUPS: CleanupEntries = {
       return {result: false, target: ERROR_TARGETS.URL};
     },
   },
+  'acum': {
+    match: [/^(https:\/\/)?nocs\.acum\.org\.il\/acumsitesearchdb\//i],
+    restrict: [LINK_TYPES.otherdatabases],
+    clean(url) {
+      return url.replace(/&.*/, '').replace('/version?', '/work?');
+    },
+    validate(url, id) {
+      const m = /^(https:\/\/)?nocs\.acum\.org\.il\/acumsitesearchdb\//i.test(url);
+      if (m) {
+        switch (id) {
+          case LINK_TYPES.otherdatabases.work:
+            return {
+              result: /\/work\?workid=[0-9A-Z]+$/.test(url),
+              target: ERROR_TARGETS.ENTITY,
+            };
+          case LINK_TYPES.otherdatabases.artist:
+            return {
+              result: /\/results\?(creatorid=I-\d+-\d|creatoripnnumber=\d+|performerid=\d+)$/.test(url),
+              target: ERROR_TARGETS.ENTITY,
+            }
+          case LINK_TYPES.otherdatabases.label:
+            return {
+              result: /\/results\?(creatorid=I-\d+-\d|creatoripnnumber=\d+)$/.test(url),
```

Comment:
Is this an actual page? It looks like a search URL, but I cannot see it since it still blocks me, so :)

Reply:
That's the page you get to when clicking a label name. Same for artist.